charts/gardener/gardenlet/values.yaml

```diff
@@ -94,6 +94,9 @@ global:
         shootStateSync:
           concurrentSyncs: 5
           syncPeriod: 30s
+      resources:
```

Comment:
Why is this part of the gardenlet configuration itself? Shouldn't it be part of the `Seed`'s specification? This way, the gardenlet uses the same resource settings for all seeds it is reconciling (although, these seeds might indeed have different capacities).

Reply:
This was discussed at length in the team meeting. The final decision was to put `capacity` and `allocatable` in the seed status, but not in the seed spec, following the `kubelet` model, as initially proposed. The new `ShootedSeed` resource would also contain the gardenlet configuration and there it's in the spec. See also the relevant sections in the GEP.

Reply:
Hm, okay, maybe I missed that because I could only join late. What were again the arguments for this decision? Just "because it's in Kubernetes/kubelet like this" is a weak one, I guess. kubelets cannot be responsible for multiple nodes, only one node, but the gardenlet can theoretically be responsible for the entire list of seeds registered in a Gardener landscape. I agree that this breaks also for other configuration options (e.g., you want to enable/disable feature gates on a per-seed basis), but at least for new fields we could try to move it to the right place.
Anyways, I won't insist here if the majority decided that this is the way to go, I just wanted to raise the topic for reconsideration.

Reply:
I think the discussion revolved around `spec` (user updatable and causes reconciliation when changed) and `status` (not user updatable and ignored by controllers on updates by other controllers). If we consider custom resources with their own extensions / plugins in the future, we were inclined to see `capacity` more of a status thing. Also, `allocatable` is now calculated based on a `reserved` configuration, but in the future `gardenlet` could try to do something smarter to determine a correct value (again, similarly to `kubelet`). The argument that other configuration options aren't configurable per seed (as you also pointed out above) also came up. We also considered adding to `spec` **and** `status`, but eventually `ShootedSeed` was proposed as the way to have both - `capacity` / `allocatable` in the Seed status but the Gardenlet configuration as part of the ShootedSeed spec, together with other values that need to be customized per Seed, and this remained as the consensus.

Reply:
OK, thank you!